### PR TITLE
Add Uppy.io

### PR DESCRIPTION
**Pull request by arturi**
I’m Russian, and I’m Uppy’s lead developer, although the project wouldn’t be possible without our team and contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@
 - **Telegram apps** by developers from [Telegram](https://telegram.org/apps)<br>
   Messenger clients for [iOS](https://github.com/TelegramMessenger/Telegram-iOS), [Android](https://github.com/DrKLO/Telegram), [macOS](https://github.com/overtake/TelegramSwift), [desktop](https://github.com/telegramdesktop/tdesktop), [web](https://github.com/zhukov/webogram). And [TDLib](https://github.com/tdlib/td).
 
+- **[Uppy](https://github.com/transloadit/uppy)** by [Artur Paikin](github.com/arturi/), the [Transloadit](https://github.com/transloadit/) team and numerous amazing contributors.<br>
```
Comment (arturi) [inline README.md:54]:
```suggestion
- **[Uppy](https://github.com/transloadit/uppy)** by [Artur Paikin](https://github.com/arturi/), the [Transloadit](https://github.com/transloadit/) team and numerous amazing contributors.<br>
```

Comment (igoradamenko) [inline README.md]: Could you please remove the dot in the end of this line?

Trying to keep the file neat and clean ^_^

Comment (igoradamenko) [reply README.md]: I will do it myself 🐨

**Comment by igoradamenko** [comment]
Thanks!

**Comment by arturi** [comment]
Thanks!

**Pull request merged**